src/components/body/multimedia.js

```diff
@@ -0,0 +1,155 @@
+import mq from '../../utils/media-query'
+import styled from 'styled-components'
+import typography from '../../constants/typography'
+
+const mockup = {
+  mobile: {
+    width: {
+      // 20px is border-right and border-left width of body
+      normal: 'calc(100% + 20px)',
+      // 355 = 375 - 20 (border width of body)
+      small: 'calc(250/355*100%)', // %
+    },
+    caption: {
+      width: 250, // px
+    },
+  },
+  tablet: {
+    width: {
+      // 20px is border-right and border-left width of body
+      normal: 'calc(100% + 20px)',
+      // 748 = 768 - 20 (border width of body)
+      small: 'calc(512/748*100%)', // %
+    },
+    caption: {
+      width: 512, // px
+    },
+  },
+  desktop: {
+    width: {
+      normal: 100, // %
+      small: 403, // px
+    },
+    caption: {
+      width: 180, // px
+    },
+  },
+  hd: {
+    width: {
+      normal: 100, // %
+      small: 532, // px
+    },
+    caption: {
+      width: 265, // px
+    },
+  },
+}
+
+export default {
+  Block: styled.div`
+    position: relative;
+
+    & > figure, & > iframe {
+      padding: 0;
+      border: 0;
+      margin: 0;
+    }
+
+    ${mq.tabletAndBelow`
+      // 10px is the left border width of body
```

Comment:
I thought `//` will not work in template literal and CSS, but it seems works fine. 👀

Reply:
I am not sure either.
However, I will change `//` to `/**/` instead.

Reply:
It seems that `styled-components` supports line comments.
https://github.com/styled-components/stylelint-processor-styled-components/issues/156